packages/widget-chat/src/containers/chat-widget/index.js

```diff
@@ -52,6 +57,12 @@ export class ChatWidget extends Component {
     return nextProps.sparkState.connected !== props.sparkState.connected || nextProps.user !== props.user || nextProps.conversation !== props.conversation;
   }
 
+  componentDidUpdate() {
+    if (this.activityList) {
```

Comment:
So, here's where it gets hairy. you only want to scroll to the bottom if the message was sent from this device
or the viewport was already at the bottom when the message came in